src/style.css

```diff
@@ -177,6 +177,23 @@ a:hover {
   transition: all 0.3s ease-out;
 }
 
+.transition-btn {
+  transition: 0.3s;
+}
+
+.break-all {
+  word-break: break-all;
+}
+
+.hero-title img {
+  width: 300px;
+  margin-bottom: 10px;
+}
```

Comment:
Can we use utility classes on the elements and try and use rem for width? https://tailwindcss.com/docs/width

Reply:
already removed this, cause don't need to using new logo